src/current/cockroachcloud/regions.md

```diff
@@ -76,39 +65,26 @@ Western Europe  | `eu-central-1`   | Frankfurt
                 | `eu-west-2`      | London
                 | `eu-west-3`      | Paris
 
-## Azure regions
+</section>
 
-CockroachDB {{ site.data.products.cloud }} clusters can be deployed in the following [Azure regions](https://learn.microsoft.com/azure/reliability/availability-zones-overview#regions):
+## GCP regions
 
-Geographic Area | Region Name          | Location
```

Comment:
I reviewed these big blocks of text moving around by pasting them locally and manually checking with `diff` and the list is unchanged (maybe obv but needs checking when stuff moves)